oap-server/server-starter/src/main/resources/ui-initialized-templates/kong/kong-service.json

```diff
@@ -0,0 +1,323 @@
+                  "i": "0",
+                  "type": "Widget",
+                  "widget": {
+                    "title": "Total bandwidth Trend (ingress/egress) (KB/s)"
+                  },
+                  "metricConfig": [
+                    {
+                      "label": "HTTP bandwidth Trend (requests per second)",
+                      "unit": "KB/s"
+                    }
+                  ],
+                  "graph": {
+                    "type": "Line",
+                    "step": false,
+                    "smooth": false,
+                    "showSymbol": true,
+                    "showXAxis": true,
+                    "showYAxis": true
+                  },
+                  "expressions": [
+                    "aggregate_labels(meter_kong_service_http_bandwidth,sum(direction))/1024"
+                  ]
+                },
+                {
+                  "x": 12,
+                  "y": 0,
+                  "w": 12,
+                  "h": 13,
+                  "i": "1",
+                  "type": "Widget",
+                  "widget": {
+                    "title": "HTTP Status Trend (requests per second)"
+                  },
+                  "metricConfig": [
+                    {
+                      "label": "HTTP Status Trend",
+                      "unit": "req/s"
+                    }
+                  ],
+                  "graph": {
+                    "type": "Line",
+                    "step": false,
+                    "smooth": false,
+                    "showSymbol": true,
+                    "showXAxis": true,
+                    "showYAxis": true
+                  },
+                  "expressions": [
+                    "aggregate_labels(meter_kong_service_http_status,sum(code))"
+                  ]
+                },
+                {
+                  "x": 0,
+                  "y": 13,
+                  "w": 12,
+                  "h": 13,
+                  "i": "2",
+                  "type": "Widget",
+                  "expressions": [
+                    "latest(aggregate_labels(meter_kong_service_nginx_metric_errors_total,sum))"
+                  ],
+                  "graph": {
+                    "type": "Card",
+                    "fontSize": 24,
+                    "textAlign": "center",
+                    "showUnit": true
+                  },
+                  "metricConfig": [
+                    {
+                      "label": "Nginx metric errors"
+                    }
+                  ],
+                  "widget": {
+                    "title": "Nginx metric errors"
+                  }
+                },
+                {
+                  "x": 12,
+                  "y": 13,
+                  "w": 12,
+                  "h": 13,
+                  "i": "3",
+                  "type": "Widget",
+                  "expressions": [
+                    "latest(aggregate_labels(meter_kong_service_datastore_reachable,sum))"
```

Comment:
This means how many reachable datastore instances are in the Service?  If so the result is not only `0/1`?

Reply:
Is it better to reserve the `instance label`?

Reply:
For an `instance`, 0/1 represents whether the database is reachable, but I will do a sum operation on the `cluster` dimension for now.

Reply:
Will we expect N connected or just 0/1 for the whole cluster?

Reply:
I think N is more representative of the current connection situation, not every instance is connected to the database.

Reply:
I mean you could show all the instances status(with labels) on the cluster(service) dashboard, do not have to sum them. I think this is more directly to show the status, the users should focus on which instance is not connected.

Reply:
> I mean you could show all the instances status(with labels) on the cluster(service) dashboard, do not have to sum them. I think this is more directly to show the status, the users should focus on which instance is not connected.

It's good, let me adjust it later.

Reply:
It will be like this
~~~
type: TIME_SERIES_VALUES
results:
- metric:
       lables:
        - key: service_instance_id
        value: kong-1
~~~
Let me use the `Table` displays the result.

Reply:
@wankai123 I think we discussed `SINGLE_VALUE` with multiple labels. Should we consider applying that for now?

Reply:
Oops, I rechecked and find that there was something wrong with `--expression` parameter, SINGLE_VALUE is correct, let me fix it.

Reply:
@CodePrometheus, I have talked with @Fine0830, she will add Value Mappings option for Card(one value, in instance dashboard) and Table(multiple value with labels, in service dashboard) widgets.

Reply:
Well received and thanks @Fine0830

Reply:
The pr about Value Mapping is [here](https://github.com/apache/skywalking-booster-ui/pull/419)